letsencrypt/cli.py

```diff
@@ -370,14 +370,16 @@ def diagnose_configurator_problem(cfg_type, requested, plugins):
                    "your existing configuration.\nThe error was: {1!r}"
                    .format(requested, plugins[requested].problem))
     elif cfg_type == "installer":
-        if os.path.exists("/etc/debian_version"):
-            # Debian... installers are at least possible
-            msg = ('No installers seem to be present and working on your system; '
-                   'fix that or try running letsencrypt with the "certonly" command')
+        if os.path.exists("/etc/debian_version") and le_util.exe_exists("apachectl"):
+            # Debian + Apache... installers are at least possible
+            msg = ('No certificate installer plugins were found on your system; '
+                   'fix that or run "letsencrypt certonly" to get a certificate '
```

Comment:
nit: Perhaps the wording should be something like `'fix that or run Let's Encrypt with "certonly" on the command line'`.

Reason for this is i doubt many people will be running the client without `letsencrypt-auto` initially, but I don't like baking `letsencrypt-auto` into the command either.